source/root.js

```diff
@@ -3,6 +3,6 @@
 import {AppRegistry, YellowBox} from 'react-native'
 import App from './app'
 
-YellowBox.ignoreWarnings([])
+YellowBox.ignoreWarnings(['Invalid prop `containerTagName`'])
```

Comment:
```suggestion
YellowBox.ignoreWarnings([
  'Failed prop type: Invalid prop `containerTagName` of type `object` supplied to `ReactMarkdown`, expected `function`',
])
```

I want to be as specific as possible with this error silencing.

I think it does some substring matching, which is why `Invalid prop` matched `Failed prop type: Invalid prop`, but I'd rather be specific.